### PR TITLE
#5 read csv with pandas

### PR DESCRIPTION
**Pull request by ImmortalChaos**

### CODE REVIEW

File: src/pyDataset.py [hunk 1]
```diff
@@ -1,35 +1,107 @@
...
+		return float(text)
+	return int(text)
+
+def splitConfigArgumentToken(txt) :
+	if txt==None or txt=="" :
+		return txt
+
+	lines = txt.split(",")
+	if len(lines)==1 : # String
+		rt = txt.split("|")
```
Comment (dodoAlice) [inline src/pyDataset.py]: Do not use split.
use 'in' keyword.

File: test/test_pyDataset.py [hunk 2]
```diff
@@ -22,10 +30,13 @@ def test_getInfo(self) :
...
 	def test_getKeys(self) :
 		dsk = self.ds.getKeys()
-		self.assertEqual(dsk[0], "AVERAGE PRICE OF KOREAN A LOT-SOLID APARTMENT")
+		self.assertEqual(dsk[0], "SAMPLE DATA")
+		self.assertEqual(dsk[1], "AVERAGE PRICE OF KOREAN A LOT-SOLID APARTMENT")
 
 	def test_loadData(self) :
-		self.assertEqual(self.ds.loadData('Average price of korean a lot-solid apartment'), "2019-01-18");
+		data = self.ds.loadData('Sample Data')
+		print(data.head())
```
Comment (dodoAlice) [inline test/test_pyDataset.py]: delete debug code!

File: src/pyDataset.py (hunk 1 above)
Comment (ImmortalChaos) [reply src/pyDataset.py]: Fixed It!

File: test/test_pyDataset.py (hunk 2 above)
Comment (ImmortalChaos) [reply test/test_pyDataset.py]: I Got it! Fixed It!

**Pull request merged**